source/patterns/@aws-solutions-constructs/aws-eventbridge-kinesisstreams/test/integ.eventbridge-kinesisstreams-new-eventbridge.ts

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
new-eventbus rather than new-eventbridge?

Reply:
fixed and updated the PR